src/main/java/org/galatea/starter/entrypoint/HalRestController.java

```diff
@@ -0,0 +1,45 @@
+package org.galatea.starter.entrypoint;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.EqualsAndHashCode;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.ToString;
+import lombok.extern.slf4j.Slf4j;
+import net.sf.aspect4log.Log;
+import net.sf.aspect4log.Log.Level;
+import org.galatea.starter.service.HalService;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * REST Controller that listens to http endpoints and allows the caller to send text to be
+ * processed
+ */
+@RequiredArgsConstructor
+@ToString
+@EqualsAndHashCode
+@Slf4j
+@Log(enterLevel = Level.INFO, exitLevel = Level.INFO)
+@RestController
+public class HalRestController {
+
+  @NonNull
+  HalService halService;
+
+  /**
+   * Send the received text to the HalService to be processed and send the result out
+   */
+  // @GetMapping to link http GET request to this method
+  // @RequestParam to take a parameter from the url
+  @GetMapping(value = "${webservice.halpath}", produces = {MediaType.APPLICATION_JSON_VALUE})
+  public String halEndpoint(@RequestParam(value = "text", required = true) String text)
+      throws JsonProcessingException {
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
You shouldn't need to manually use the ObjectMapper here.

Reply:
- [ ] Remove need to use ObjectMapper manually

Reply:
Without the manual call to the object mapper the returned string doesn't get converted into JSON. Looking at POSTMAN I get "Unexpected Token 'T'" for the response